analysis_options.yaml

```diff
@@ -85,3 +85,4 @@ linter:
     - use_string_buffers
     - use_super_parameters
     - use_to_and_as_if_applicable
+    - unreachable_from_main
```

Comment:
Whoops! Could you sort this up? (It should go before `use_if_null_to_convert_nulls_to_bools` I think?)

Reply:
Oops. You're super right, thanks!